go.mod

```diff
@@ -3,69 +3,52 @@ module github.com/giantswarm/cluster-operator
 go 1.16
 
 require (
-	github.com/beorn7/perks v1.0.1 // indirect
-	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/giantswarm/apiextensions v0.1.1
+	github.com/giantswarm/apiextensions v0.2.0
 	github.com/giantswarm/apprclient v0.0.0-20191209123802-955b7e89e6e2
-	github.com/giantswarm/backoff v0.0.0-20200209120535-b7cb1852522d
-	github.com/giantswarm/certs v0.0.0-20191209164338-3f774da07e59
+	github.com/giantswarm/backoff v0.2.0
+	github.com/giantswarm/certs/v2 v2.0.0
 	github.com/giantswarm/clusterclient v0.0.0-20200127145418-6c6f565f94c7
 	github.com/giantswarm/errors v0.0.0-20200227191412-38679efaafe8
-	github.com/giantswarm/exporterkit v0.0.0-20190619131829-9749deade60f // indirect
-	github.com/giantswarm/k8sclient v0.0.0-20200120104955-1542917096d6
+	github.com/giantswarm/k8sclient v0.2.0
 	github.com/giantswarm/kubeconfig v0.0.0-20191209121754-c5784ae65a49
 	github.com/giantswarm/microclient v0.0.0-20190809131213-459b479d046f // indirect
-	github.com/giantswarm/microendpoint v0.0.0-20191121160659-e991deac2653
-	github.com/giantswarm/microerror v0.1.0
-	github.com/giantswarm/microkit v0.0.0-20191023091504-429e22e73d3e
-	github.com/giantswarm/micrologger v0.1.1
-	github.com/giantswarm/operatorkit v0.0.0-20200309100035-748c399a1bba
+	github.com/giantswarm/microendpoint v0.2.0
+	github.com/giantswarm/microerror v0.2.0
+	github.com/giantswarm/microkit v0.2.0
+	github.com/giantswarm/micrologger v0.3.1
+	github.com/giantswarm/operatorkit v0.2.1
 	github.com/giantswarm/resource v0.0.0-20201203121554-9ca9ddbde7cf
-	github.com/giantswarm/tenantcluster v0.0.0-20191209135534-800974a4d4bf
-	github.com/giantswarm/to v0.2.0 // indirect
-	github.com/giantswarm/versionbundle v0.0.0-20200203095303-cd94540b7d5a
-	github.com/go-logfmt/logfmt v0.5.0 // indirect
+	github.com/giantswarm/tenantcluster/v2 v2.0.0
```

Comment:
This one seems wacky. The changelog in the project is broken so the included changes are unclear to me.

Reply:
Yeah, I looked at the diff and it looks to be entirely dependency bumps similar to this one. Unfortunately neither the release before nor the one after will work